docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -0,0 +1,48 @@
+html {
+  font-size: 16px;
+
+  @include colorful {
+    font-family: $colorful-font--sans;
+  }
+}
+
+h1 {
+  @include hb_heading-1;
+}
+
+h2 {
+  @include hb_heading-2;
+}
+
+h3 {
+  @include hb_heading-3;
+}
+
+h4 {
+  @include hb_heading-4;
+}
+
+h5 {
+  @include hb_heading-5;
+}
+
+h6 {
+  @include hb_heading-6;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  a {
+    @include hb_link;
+  }
+}
+
+p {
```

Comment:
We don't need to wrap the anchor tag in the paragraph here. There are a lot of other scenarios where we'll want an inline anchor tag that isn't inside a paragraph. This should just be the default styles (with the `:not([class])` specification mentioned above).